README.rst

```diff
@@ -151,10 +203,96 @@ display add the new tab for On Task:
         },
     }
 
-You can add it using your favorite configuration method. Then, you'll see:
+It is also necessary to include these settings for each course using the
+**Other Course Settings** in the **Advanced Settings** section in Studio:
+
+- **ONTASK_API_AUTH_TOKEN** *(Required)*: API Auth token for the OnTask
+  service. You can find it in the OnTask service from **Profile** >
+  **API Auth token**. If the token has not been generated, you can create
+  a new one.
+- **ONTASK_WORKFLOW_ID** *(Optional)*: ID of the workflow in OnTask. If you
+  already have a workflow, you can include it here. If you do not have a
+  workflow, the plugin will create a new one from the OnTask tab in the
+  instructor dashboard.
+
+Example:
+
+.. code-block:: json
+
+  {
+    ...
+    "ONTASK_API_AUTH_TOKEN": "your-api-auth-token",
+    "ONTASK_WORKFLOW_ID": 1
+  }
+
+**NOTE**: It is important to have enabled the **Other Course Settings** in the
+settings of the platform. You can find more information about this in the
+`Open edX documentation`_.
+
+.. _Open edX documentation: https://edx.readthedocs.io/projects/edx-installing-configuring-and-running/en/latest/configuration/enable_custom_course_settings.html
+
+
+View from the Learning Management System (LMS)
+**********************************************
+
+When the instructor accesses the OnTask tab, they will have the option to
+create an OnTask Workflow in case there is not one configured. Once the
+**Create Workflow** button is clicked, a workflow will be created in OnTask and
+will be named as the current course ID. e.g. ``course-v1:edunext+ontask+demo``.
+
+.. image:: https://github.com/eduNEXT/platform-plugin-ontask/assets/64033729/ab2c2623-1b74-479e-b8f2-7d37cb4b9a64
+
+The instructor can then view the OnTask workflow related to the course. From
+there the instructor can create, import/export or execute all actions. In the
+upper left corner there will be a button called **Load Data**. Once pressed, an
+asynchronous task will be executed that will generate a data summary of the
+course and load it into a table in the OnTask Workflow. Each time the button is
+pressed, the data will be overwritten. The instructor can then use this data to
```

Comment:
Is _Each time the button is pressed, the data table will be recreated_ a better description?

Reply:
Done: https://github.com/eduNEXT/platform-plugin-ontask/commit/69d1b4881df347721a718e87f6ee1ff68c598001